### PR TITLE
chapter 2 proofreading

### PR DESCRIPTION
**Pull request by larskotthoff**

### CODE REVIEW

File: book/chapters/chapter2/data_and_basic_modeling.qmd [hunk 1]
```diff
@@ -97,7 +97,8 @@ The `id` argument is optional and specifies an identifier for the task that is u
 ## UTF8 Column Names
 
 As many machine learning models do not work properly with arbitrary UTF8 names, `r mlr3` defaults to throwing an error if any of the column names passed to `r ref("as_task_regr()")` (and other task constructors) contain a non-ASCII character or do not comply with R's variable naming scheme.
-Therefore, we recommend converting names with `r ref("make.names()")` if possible, but if not then you can bypass this check by setting `options(mlr3.allow_utf8_names = TRUE)` (but do not be surprised if an underlying package implementation throws up a related error).
+Therefore, we recommend converting names with `r ref("make.names()")` if possible.
+If you know what you are doing, you can bypass this check by setting `options(mlr3.allow_utf8_names = TRUE)` (but do not be surprised if errors occur).
```
Comment (RaphaelS1) [inline book/chapters/chapter2/data_and_basic_modeling.qmd:101]:
```suggestion
You can bypass this check by setting `options(mlr3.allow_utf8_names = TRUE)` (but do not be surprised if errors occur).
```

File: book/chapters/chapter2/data_and_basic_modeling.qmd [hunk 2]
```diff
@@ -562,7 +563,7 @@ as.data.table(msr())
 All measures implemented in `mlr3` are defined primarily by three components: 1) the function that defines the measure; 2) whether a lower or higher value is considered 'good'; and 3) the range of possible values the measure can take.
 As well as these defining elements, other metadata are important to consider when selecting and using a `Measure`, including if the measure has any special properties (e.g., requires training data), the type of predictions the measure can evaluate, and whether the measure has any 'control parameters'.
 All this information is encapsulated in the `r ref("Measure", aside = TRUE)` object.
-By example, let us consider the `r index('mean absolute error', parent = "measures")` (MAE):
+For example, let us consider the `r index('mean absolute error', parent = "measures")` (MAE):
```
Comment (RaphaelS1) [inline book/chapters/chapter2/data_and_basic_modeling.qmd:566]: By example is correct. We are demonstrating something by an example

Comment (RaphaelS1) [inline book/chapters/chapter2/data_and_basic_modeling.qmd:566]:
```suggestion
By example, let us consider the `r index('mean absolute error', parent = "measures")` (MAE):
```

File: book/chapters/chapter2/data_and_basic_modeling.qmd [hunk 3]
```diff
@@ -581,13 +582,13 @@ All supervised learning measures compare the difference between predicted values
 `mlr3` simplifies the process of bringing these quantities together by storing the predictions and true outcomes in the `r ref("Prediction", index = TRUE)` object as we have already seen.
 
 ```{r}
-prediction
+predictions
```
Comment (RaphaelS1) [inline book/chapters/chapter2/data_and_basic_modeling.qmd:585]: Marc changed predictions -> prediction everywhere as part of unifying book terminology. I'm not saying change back but this chapter will now be out of sync with others

File: book/chapters/chapter2/data_and_basic_modeling.qmd (hunk 2 above)
Comment (larskotthoff) [reply book/chapters/chapter2/data_and_basic_modeling.qmd]: Yes, but it sounds weird.

File: book/chapters/chapter2/data_and_basic_modeling.qmd (hunk 3 above)
Comment (larskotthoff) [reply book/chapters/chapter2/data_and_basic_modeling.qmd:585]: I find the name `predictions` more intuitive as there's clearly more than one of them, but I'll leave it up to you whether to change it again.

File: book/chapters/chapter2/data_and_basic_modeling.qmd (hunk 2 above)
Comment (RaphaelS1) [reply book/chapters/chapter2/data_and_basic_modeling.qmd]: Sure but it's grammatically correct and we've chosen formal writing everywhere

File: book/chapters/chapter2/data_and_basic_modeling.qmd (hunk 3 above)
Comment (RaphaelS1) [reply book/chapters/chapter2/data_and_basic_modeling.qmd:585]: @be-marc how difficult would it be to update this everywhere? / how much time would it take?

**Pull request merged**